src/Currency.php

```diff
@@ -86,12 +86,16 @@ public function convert($amount, $from = null, $to = null, $format = true)
             return null;
         }
 
-        // Convert amount
-        if ($from === $to) {
-            $value = $amount;
-        }
-        else {
-            $value = ($amount * $to_rate) / $from_rate;
+        try {
+            // Convert amount
+            if ($from === $to) {
+                $value = $amount;
+            } else {
+                $value = ($amount * $to_rate) / $from_rate;
+            }
+        } catch (\Exception $e) {

```

Comment:
```suggestion
        } catch (ArithmeticError $e) {
```
it's always better to catch a specific exception instead of the most generic one, catching the generic exception can lead to having the most annoying bugs to debug since it can mask the real exception and suppress it.